docs/en/latest/apisix.md

```diff
@@ -31,6 +31,18 @@ helm repo update
 helm install apisix apisix/apisix --create-namespace  --namespace apisix
 ```
 
+### APISIX v3.x
+
+The current default version of APISIX installed is 2.15.0, if you want to install APISIX 3.x, to install the chart of APISIX with:
+
+```shell
+helm install apisix apisix/apisix --set apisix.image.tag=3.0.0-debian --create-namespace --namespace apisix
+```
+
+:::note
+
+apisix-ingress-controller is not compatible with APISIX 3.x.
```

Comment:
the dashboard is also not compatible with APISIX 3.x, right?

Reply:
updated